src/main/java/org/folio/circulation/domain/LoanService.java

```diff
@@ -0,0 +1,51 @@
+package org.folio.circulation.domain;
+
+import java.util.Collection;
+import java.util.concurrent.CompletableFuture;
+
+import org.folio.circulation.domain.policy.LoanPolicy;
+import org.folio.circulation.domain.policy.library.ClosedLibraryStrategyService;
+import org.folio.circulation.support.Clients;
+import org.folio.circulation.support.Result;
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeZone;
+
+import static java.util.concurrent.CompletableFuture.completedFuture;
+import static org.folio.circulation.support.Result.succeeded;
+
+public class LoanService {
```

Comment:
This name is very general. Is there an intent for more logic to move here?

Reply:
Yes, class was created as a general service for loans.
Do you want me to rename it to be more specific?

Reply:
I'm only trying to understand the context and whether the intent is to include other aspects in this.

The only method that is there is specific to applying the impact of a loan policy when there are requests present.

Reply:
Method itself works with recalls still class was created to store (in future) business logic related to loans.

Reply:
What aspects of loan business logic do you envisage might belong in this class?

Reply:
I didn't think about some specific business logic, just realized that there was no service to add business logic for loans